src/main/java/app/model/agents/Grid.java

```diff
@@ -84,4 +85,22 @@ public double getCellSize()
     {
         return cellSize;
     }
+
+    public int getCellsExplored()
+    {
+        // Loop through grid and divide explored cells by total cells
+        int cellsExplored = 0;
+        for(Cell[] row : grid)
+        {
+            for(Cell cell : row)
+            {
+                BooleanCell bc = (BooleanCell) cell;
```

Comment:
Will need to accommodate for Pheromone cell as well as BooleanCell, as the ACO agent only uses Pheromone cell.